### PR TITLE
Document client.queryWatermarkOffsets

### PR DESCRIPTION
**Pull request by atamon**
I found myself digging quite long for a solution to this problem.

According to https://github.com/Blizzard/node-rdkafka/pull/170/commits/77bbb7527ba6a46cbfa455b6af6c1275cb43f498#diff-50cfa59973c04321b5da0c6da0fdf4feR441, I believe something like the above would be a correct way of calling the function.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -411,6 +411,25 @@ The following table lists events for this API.
 |`event.stats` | The `event.stats` event is emitted when `librdkafka` reports stats (if you opted in). |
 |`event.throttle` | The `event.throttle` event is emitted when `librdkafka` reports throttling.|
 
+## Reading current offsets from the broker for a topic
+
+Some times you find yourself in the situation where you need to know the latest (and earliest) offset for one of your topics. Connected producers and consumers both allow you to query for these through `queryWaterMarkOffsets` like follows:
+
+```js
+const timeout = 5000, partition = 0;
```
Comment (webmakersteve) [inline README.md]: Please change these `const`s to `var`s, just to be consistent among the rest of the README.

File: README.md [hunk 2]
```diff
@@ -411,6 +411,25 @@ The following table lists events for this API.
...
+consumer.queryWatermarkOffsets('my-topic', partition, timeout, function(err, offsets) {
+  const high = offsets.highOffest;
+  const low = offsets.lowOffset;
+});
```
Comment (webmakersteve) [inline README.md]: Can you add a newline here?

**Comment by webmakersteve** [comment]
Thanks for the PR. It wasn't in the README because it is not critical for node-rdkafka's use, and there is a separate page for complete documentation. But I'm never against anything additional.

Can you make those two small changes for me? Then I'd be happy to merge.

**Comment by atamon** [comment]
I fixed the comments.

And wow, I can't believe how stupid I feel. I didn't find the docs at https://blizzard.github.io/node-rdkafka/current/index.html yesterday.

Thanks for enlightening me! And once again, great work!

**Comment by webmakersteve** [comment]
Merged! Thanks for the PR!

And no problem! They're harder to find than I wish they were :P

**Pull request merged**